src/dmd/mars.d

```diff
@@ -769,6 +749,56 @@ private int tryMain(size_t argc, const(char)** argv, ref Param params)
             }
         }
     }
+
+    // Output the makefile dependencies
+    if (params.emitMakeDeps)
+    {
+        OutBuffer* buf = new OutBuffer();
+
+        // start by resolving and writing the target (which is sometimes resolved during link phase)
+        if (params.link && params.exefile)
+        {
+            escapeMakePath(buf, &params.exefile[0]);
```

Comment:
I suggest using UFCS for better readability (it's easier to follow the dataflow):
```suggestion
            buf.writeMakePath(params.exefile);
```